scapy/layers/isakmp.py

```diff
@@ -409,10 +409,16 @@ class ISAKMP_payload_SA(ISAKMP_payload):
 
 class ISAKMP_payload_Nonce(ISAKMP_payload):
     name = "ISAKMP Nonce"
+    fields_desc = ISAKMP_payload.fields_desc[:3] + [
```

Comment:
Hi. Sorry for the delay.

This looks okay but will break existing code. You should add a `deprecated_fields` (grep elsewhere in the code) to redirect the old 'load' to those newer names.

Thanks